File: poc-kronos-frontend/src/model/comum/Cidade.js
class Cidade {
  constructor(dados) {
    this.id = dados ? dados.id : null;
    this.nome = dados ? dados.nome : null;
    this.area = dados ? dados.area : null;
    this.codAreaTel = dados ? dados.codAreaTel : null;
    this.codigoIbge = dados ? dados.codigoIbge : null;
    this.idEstado = dados ? dados.idEstado : null;
  }
}
export default Cidade;

File: src/main/java/br/com/kronos/backend/aplicacao/comum/Cidade.java
package br.com.kronos.backend.aplicacao.comum;

import java.io.Serializable;

import lombok.Builder;
import lombok.Data;
import lombok.EqualsAndHashCode;

@EqualsAndHashCode(callSuper=false)
@Builder
@Data
public class Cidade implements Serializable {
	
	private static final long serialVersionUID = 1L;
	private Integer id;
	private String nome;
	private String area;
	private String codigoIbge;
	private String codAreaTel;
	private Integer idEstado;

}
